backend/src/near.js

```diff
@@ -16,10 +21,37 @@ const queryFinalTimestamp = async () => {
 };
 
 const queryNodeStats = async () => {
-  let nodes = await nearRpc.sendJsonRpc("validators", [null]);
-  let proposals = nodes.current_proposals;
-  let currentValidators = getCurrentNodes(nodes);
-  return { currentValidators, proposals };
+  let genesisConfig = await nearRpc.sendJsonRpc(
```

Comment:
It is not a genesis config anymore, we use the latest protocol config for the network, so it makes sense to name it `networkProtocolConfig`:

```suggestion
  let networkProtocolConfig = await nearRpc.sendJsonRpc(
```